app.js

```diff
@@ -0,0 +1,18 @@
+const { getTopics } = require("./controller");
+const express = require("express");
+
+console.log("app invoked");
+const app = express();
+
+app.get("/api/topics", getTopics);
+
+app.use((req, res, next) => {
+  res.status(404).send({ msg: "Path not found" });
+});
+
+app.use((err, req, res, next) => {
```

Comment:
We shouldn't need this

Reply:
For your 404 error, you may look to use a generic 404 error, so we can be sure we only fall into that block of code if there is an invalid path (look at app.all and using a wildcard * )

https://notes.northcoders.com/courses/js-back-end/error-handling#5-generic-404-errors